terraform/aws/modules/bind-server/bind-servers.tf

```diff
@@ -53,20 +53,27 @@ resource "aws_key_pair" "bind" {
   public_key = var.ssh_key_public
 }
 
-resource "aws_launch_configuration" "bind_lauch_configuration" {
+resource "aws_launch_configuration" "bind_launch_configuration" {
```

Comment:
@stafot  bare in mind that when you run this it will delete the resource and recreate it.

Reply:
@stylianosrigas Launch configuration is ephemeral. If it was on ASG i would probably choose to live with the typo.